build/common.gypi

```diff
@@ -9,6 +9,16 @@
       'tizen%': '<(tizen)',
       'tizen_mobile%': '<(tizen_mobile)',
       'enable_murphy%': '<(enable_murphy)',
+      # Whether to enable WebCL.
+      'enable_webcl%': '<(enable_webcl)',
+      'variables': {
```

Comment:
The non-Android bots failed because you only set `enable_webcl` to any value on Android. I suggest a structure more similar to Chromium's `common.gypi`:

```python
{
  'variables': {
    # ...
    'conditions': [
      ['OS=="android"', {
        'enable_webcl%': 1,
      }, {
        'enable_webcl%': 0,
      }],
    ],
  },
  'target_defaults': {
    'variables': {
      # ...
      'enable_webcl': '<(enable_webcl)',
    },
    # ...
  },
}
```